database/src/main/scala/cromwell/database/SqlDatabase.scala

```diff
@@ -222,11 +220,15 @@ trait SqlDatabase extends AutoCloseable {
     *
     * @param startMetadataId        The minimum ID an entry in `METADATA_JOURNAL` must have to be examined for summary.
     * @param startMetadataTimestamp An optional timestamp.  If specified, a metadatum must have a timestamp greater than or equal to this value.
+    * @param getUpdatedSummary      Takes in the optional existing summary and the metadata, returns the new summary.
+    *
     * @return A `Future` with the maximum ID value of the metadata summarized by the invocation of this method.
     */
-  protected def refreshMetadataSummaries(startMetadataId: Long, startMetadataTimestamp: Option[OffsetDateTime])
+  protected def refreshMetadataSummaries(startMetadataId: Long, startMetadataTimestamp: Option[Timestamp],
```

Comment:
Just curious as I'm still trying to figure out what we should/shouldn't be doing with the time stuff, and you're both the person who made the change and the one who knows best - why the change from OffsetDateTime to Timestamp?